campaigns.json

```diff
@@ -986,15 +986,15 @@
   },
   "Run3Summer22NanoAOD" :{
         "fractionpass": 0.97,
```

Comment:
Why 0.97?

Reply:
The PDMV people always want the NanoAOD to be basically 100 of miniAOD because it is not run in the same chain as Gen-MiniAOD it is run after

Reply:
There are nano campaigns whose fractionpass is .95 and .99. I always prefer the lower one if possible as it'll reduce our manual work. If no failure happens they will complete 100% anyways. This fractionpass defines the threshold below which we're going to do manual recovery. Can you please investigate the possibility of lowering to .95 by contacting PDMV? If it's not okay, we can keep it as .97%

Reply:
Ok will go ask them. To open the campaign I am sure they will say fine.

Reply:
I pushed the update this is holding up other campaigns in here they want.

Reply:
I would point out that maybe we can just get ride of this because we never use it and never want to use it?